packages/ui/package.json

```diff
@@ -104,6 +108,7 @@
     "prettier": "^2.8.4",
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
+    "react-docgen-typescript": "~2.2.2",
```

Comment:
This lib needs to be `dev dependancy`. Please check any other dependancies as well.. We have added them as dependancies for no reason.